src/general/ModInfo.cs

```diff
@@ -0,0 +1,16 @@
+using Godot;
+using Newtonsoft.Json;
+
+public class ModInfo : Node
+{
+    [JsonProperty("Name")]
+    public string ModName;
```

Comment:
Why does this property have different name in the json? Why not just have `public string Name` ?

Reply:
Calling it Name gave me a warning so i just changed it to get rid of it

Reply:
I see that this inherits from Node. There doesn't seem a reason for that. It will probably work if you don't inherit this class from anything.